nixos/tests/forked-daapd.nix

```diff
@@ -0,0 +1,44 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  configDir = "/var/lib/foobar";
```

Comment:
`foobar` is not a helpful name. I think you only specify it to refer to it later, but it would still be nice to have something like `"/var/lib/forked-daapd"` here.

Reply:
Fixed